Tutorial Series/Introduction to Financial Python/Tutorial10 Multiple Linear Regression.html

```diff
@@ -1,102 +1,210 @@
+        <td align="right">-0.000719</td>
+        <td align="right">0.001976</td>
+    </tr>
+    <tr>
+        <td>2016-12-27</td>
+        <td align="right">0.002254</td>
+        <td align="right">0.014113</td>
+        <td align="right">0.014993</td>
+        <td align="right">0.002298</td>
+        <td align="right">0.006331</td>
+    </tr>
+    <tr>
+        <td>2016-12-28</td>
+        <td align="right">-0.008218</td>
+        <td align="right">0.000946</td>
+        <td align="right">-0.007635</td>
+        <td align="right">-0.005611</td>
+        <td align="right">-0.004273</td>
+    </tr>
+    <tr>
+        <td>2016-12-29</td>
+        <td align="right">-0.000247</td>
+        <td align="right">-0.009081</td>
+        <td align="right">-0.001000</td>
+        <td align="right">-0.000722</td>
+        <td align="right">-0.000257</td>
+    </tr>
+    <tr>
+        <td>2016-12-30</td>
+        <td align="right">-0.004601</td>
+        <td align="right">-0.020172</td>
+        <td align="right">-0.009720</td>
+        <td align="right">-0.002023</td>
+        <td align="right">-0.007826</td>
+    </tr>
+</table>
+
+Now let's build a model. Recall that we can perform a simple linear regression model using the 'statsmodels' package by inputing the formula:
+
+<pre class="prettyprint linenums">simple = sm.ols(formula = 'amzn ~ spy',data = df).fit()
+print simple.summary()
+
+                 coef    std err          t      P>|t|      [0.025      0.975]
+------------------------------------------------------------------------------
+Intercept   9.876e-05      0.001      0.097      0.923      -0.002       0.002
+spy            1.0796      0.124      8.725      0.000       0.836       1.323
+</pre>
+
+Similarly, we can build a multiple linear regression model:
+
+<pre class="prettyprint linenums">model = sm.ols(formula = 'amzn ~ spy+ebay+wal',data = df).fit()
+print model.summary()
+
+                 coef    std err          t      P>|t|      [0.025      0.975]
+------------------------------------------------------------------------------
+Intercept      0.0001      0.001      0.134      0.894      -0.002       0.002
+spy            1.0468      0.170      6.155      0.000       0.712       1.382
+ebay          -0.0795      0.058     -1.364      0.174      -0.194       0.035
+wal           -0.0865      0.089     -0.976      0.330      -0.261       0.088
+aapl           0.1529      0.084      1.831      0.068      -0.012       0.317
+</pre>
+
+As seen from the summary table, the P-values for Ebay, Walmart and Apple are 0.174, 0.330 and 0.068 respectively, so none of the them are significant at a 95% confidence.
+
+The multiple regression model has a higher R-square than the simple one: 0.254 vs 0.234. Indeed, R-square cannot decrease as the number of variables increases. This is because if additional variables do not explain the response (amzn), then their estimated coefficients will be zero, without losing explanatory power.

```

Comment:
"Indeed, R-square cannot decrease as the number of variables increases. This is because if additional variables do not explain the response (amzn), then their estimated coefficients will be zero, without losing explanatory power."

This can be worded better. Avoid triple negatives **" cannot... decrease... do not explain..."**